### DISCUSSION TITLE
what should the bark coding url be?

### DISCUSSION THREAD

**Discussion by Rubiidev-IPad** [cytrinc/bark-coding#182] [Polls]
​

**Comment by Rubiidev-IPad** [comment]
btw i won't be voting so have fun

**Reply by Rubiidev-IPad** [reply]
nvm

**Comment by mariocraft987** [comment]
I was going for https://bark.dev 😢

**Reply by raynepaws** [reply]
sadly, it's taken. what's really annoying, though, is that they're just hanging on to it and [not even using it](https://bark.dev/)

**Reply by mariocraft987** [reply]
☠️☠️☠️☠️☠️☠️☠️☠️☠️☠️☠️☠️☠️☠️☠️☠️☠️☠️☠️☠️

**Reply by mariocraft987** [reply]
https://bark.coding?

**Reply by mariocraft987** [reply]
Nvm, its already taken

**Reply by raynepaws** [reply]
> bark.org would be nice

when typing that link in the address bar, godaddy tells me it might be for sale

**Reply by nmsderp** [reply]
don't use godaddy, they raise prices constantly and is extremely expensive compared to others, namecheap would be best for us

**Reply by nmsderp** [reply]
if we use bark.nmsderp.xyz it would be free since i own that domain

**Reply by Rubiidev-IPad** [reply]
bark.nmsderp.xyz should be temp, but for now, bark.nmsderp.xyz it is!

**Reply by nmsderp** [reply]
@mariocraft987 has to make that decision

**Reply by nmsderp** [reply]
and he has to set it up

**Reply by mariocraft987** [reply]
k

**Reply by mariocraft987** [reply]
which websiet?

**Reply by nmsderp** [reply]
go to bark on vercel go to domains add bark.nmsderp.xyz tell me what you see

**Reply by mariocraft987** [reply]
"Domain is pending verification"

**Reply by mariocraft987** [reply]
"Another Vercel account is using this domain. Set the following TXT record on _vercel.nmsderp.xyz to use bark.nmsderp.xyz in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed."

**Reply by nmsderp** [reply]
> "Another Vercel account is using this domain. Set the following TXT record on _vercel.nmsderp.xyz to use bark.nmsderp.xyz in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed."

nevermind i can't do this, it might give you access to my dns

**Reply by dumorando** [reply]
yoyoyoyo who wants a bark.dumorando.com domain boys.... or bark.ba4x.pro... cant do dumo.se because
![image](https://github.com/mariocraft987/bark-coding/assets/144301348/f93f0c2b-d2a1-4106-9de2-dae11cf77847)

**Reply by Rubiidev-IPad** [reply]
bark.ba4x.pro sounds coolio!

**Reply by raynepaws** [reply]
> bark.ba4x.pro sounds coolio!

hard to remember though

**Reply by dumorando** [reply]
barkcoding.com would absolutely be possible i just dont know if i have enough money for these domains
